package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fulfillment",
-  "version": "2.4.2",
+  "version": "2.4.3",
```

Comment:
We do not update the app version for each change, revert this change

Reply:
I have changed the version.